### ISSUE TITLE
Datadog policy includes S3 GetObject?

### ISSUE THREAD

**Issue by LuigiClemente-Awin** [cloudposse/terraform-aws-datadog-integration#36]
Hi all

I see that in the "all" policy attached to the Datadog role has been added the s3:GetObject action:
https://github.com/cloudposse/terraform-aws-datadog-integration/blob/c70209d806ad517890bf9e345573eb735f625088/all.tf#L64

I do not understand why it should be needed and it is not listed in the official Datadog documentation:
https://docs.datadoghq.com/integrations/amazon_web_services/?tab=roledelegation

```
                "s3:GetBucketLogging",
                "s3:GetBucketLocation",
                "s3:GetBucketNotification",
                "s3:GetBucketTagging",
                "s3:ListAllMyBuckets",
                "s3:PutBucketNotification",
```

Is this a typo?
Same for s3:ListObjects, but the security concern about GetObject seems greater in my opinion.

Thanks,

Luigi

**Comment by joe-niland** [comment]
Fixed in #46

**Comment by green3g** [comment]
It seems that s3:GetObject was re-added recently to the permissions. Is this expected, and is the solution now to manually filter that permission out?
